tvb_build/package_for_pip_client.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+
+cd ..
+
+rm -Rf dist
+mkdir dist
+
+declare -a folders2pack=("framework_tvb" "scientific_library" "externals/tvb_gdist")
+if [[ "$1" != "" ]]; then
+    echo "Received param: " "$1"
+    folders2pack=("$1")
+fi
+
+
+for pipPackage in "${folders2pack[@]}"; do
+
+    echo "============================="
+    echo " Packing: " $pipPackage
+    echo "============================="
+
+    cd $pipPackage
+    python setup_client.py sdist
+    python setup_client.py bdist_wheel
```

Comment:
These 2 lines I expect, but the loop over folders2pack does not make any sense.
Have you considered to simply add the setup_client.py use in the already existent SH (package_for_pip) ?

Reply:
That's true. The loop over folders2pack should be removed. 
For second question... Yes, I took into consideration this solution too, but I think that probably we want to package the rest-client independently of the entire framework and having two different SH for packaging allow us to do this.

Reply:
We rather want to have one single Jenkins build doing all Pypi packages. With any new release in the future I hope we can release everything at the same time